web-client/src/components/AboutInfo.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import '../App.css';
+
+function AboutInfo(props) {
+    const styles = makeStyles({
+        main:
+        {
+            padding: '5%',
+            marginBottom: '20%',
+            width: '80%',
+            margin: 'auto',
+            fontSize: '105%',
+        },
+        left:
+        {
+            width: '48%',
+            float: 'left',
+            textAlign: 'justify',
+        },
+        right:
+        {
+            width: '48%',
+            float: 'right',
+        },
+        leftphoto:
+        {
+            width: '48%',
+            float: 'left',
+        },
+        righttext:
+        {
+            width: '48%',
+            float: 'right',
+            textAlign: 'justify',
+        },
+        photo:
+        {
+            float: 'left',
+            width: '100%',
+        },
+
+    });
+
+    const style = styles();
+
+    if(props.isImageRight === true)
```

Comment:
You don't have to compare isImageRight to true (it will still be working properly) or am I missing something?